PAWS/Study Information/EnrollmentGroup.swift

```diff
@@ -0,0 +1,39 @@
+//
+// This source file is part of the PAWS application based on the Stanford Spezi Template Application project
+//
+// SPDX-FileCopyrightText: 2023 Stanford University
+//
+// SPDX-License-Identifier: MIT
+//
+
+import FirebaseAuth
+import Spezi
+import SpeziAccount
+import SpeziFirebaseAccount
+import SwiftUI
+
+
+@Observable
+class EnrollmentGroup: Module, EnvironmentAccessible {
+    @ObservationIgnored @Dependency var accountConfiguration: FirebaseAccountConfiguration?
+    @ObservationIgnored @Environment(Account.self) var account
```

Comment:
We unfortunately won't be able to access the account at runtime here as `@Environment` only works within a SwiftUI view.

We will have to manually pull the data of birth from the Firestore. As noted in the meeting we should ideally use a snapshot listener to ensure that we get informed about changes within the date of birth and if it was initially set:
https://firebase.google.com/docs/firestore/query-data/listen#swift